controllers/flowlogspipeline/flp_test.go

```diff
@@ -96,7 +96,8 @@ func getConfig() flowslatest.FlowCollectorSpec {
 			},
 		},
 		Loki: flowslatest.FlowCollectorLoki{
-			URL: "http://loki:3100/",
+			Enable: pointer.Bool(true),
```

Comment:
here also a test that switch it off changes the pipeline

Reply:
Done in https://github.com/netobserv/network-observability-operator/pull/354/commits/36eb0f243f6e43633a3731d7a313fb5feb1a1b69